systemlink/clients/tag/_buffered_tag_writer.py

```diff
@@ -197,6 +197,7 @@ def __exit__(
         if self._closed:
             return False
 
+        self.send_buffered_writes()
         self._timer_generation += 1
```

Comment:
We should be able to remove this (and the one in `__aexit__` below) since `send_buffered_writes` will do the same thing.
```suggestion
```

Reply:
Done.

Reply:
Thanks